bundles/org.openhab.binding.fmiweather/src/main/java/org/openhab/binding/fmiweather/internal/discovery/CitiesOfFinland.java

```diff
@@ -22,7 +22,7 @@
  * Cities of Finland
  *
  * Parsed from
- * https://opendata.fmi.fi/wfs?service=WFS&version=2.0.0&request=getFeature&storedquery_id=fmi::forecast::hirlam::surface::cities::multipointcoverage
+ * https://opendata.fmi.fi/wfs?service=WFS&version=2.0.0&request=getFeature&storedquery_id=fmi::forecast::harmonie::surface::cities::multipointcoverage
```

Comment:
Both URLs are not wrong. There seems to be no similar "stored query" cities.

Checking https://en.ilmatieteenlaitos.fi/open-data-manual-fmi-wfs-services we can find forecast for Finnish cities 

https://opendata.fmi.fi/wfs?service=WFS&version=2.0.0&request=getFeature&storedquery_id=ecmwf::forecast::surface::cities::multipointcoverage

But the list is different compared to currently hard-coded list.

Perhaps we just refer to old hirlam URL and mention that the resource is no longer working?